supremo/1.0.0/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM continuumio/miniconda3:23.9.0-0
+LABEL maintainer="Daniel Miller (millerd15@chop.edu)"
+LABEL descrption="SuPreMo - Pipeline for scoring simple and structural variants using Akita"
+
+RUN apt update \
+    && apt install -y build-essential curl
+
+WORKDIR /opt
+
+RUN conda install --name base python=3.10 numpy scipy pandas jupyter tensorflow Cython
```

Comment:
Conda eh? Guessing that's the author rec

Reply:
Yeah. Not a fan but I guess its just the way things are heading now with bio people loving conda.